assignment/HW_Index.html

```diff
@@ -32,48 +32,78 @@
     </script>
     <script>
       /* =====================
-
       # Week 2 - Assignment
-
       ## Introduction
-
        This week's assignment as having two separate parts:
-
        1. Log a series of arrays to the console that represents the health_centers dataset
        in array form. For example, [{'first': 1, 'second': 44}, {'first': 2, 'second': 12}]
        becomes [['first','second'],[1, 44],[2, 12]]
-
        2. Put a marker on the map for each health center in the specified lat/lng coordinates
        with a popup (a simple text dialog) that shows that location's name when its marker
        is clicked.
-
        EXTRA: Some of these locations offer dental services as well. Can you find some way to
        give dental locations a different icon? (Here's some relevant documentation:
        http://leafletjs.com/reference.html#icon)
-
        Remember: functions are meant to facilitate code comprehension and reuse. Try to
        use them to organize your code.
       ===================== */
-
-
       /* =====================
-
       Start code
-
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
-
-
       var addMarkers = function(map) {};
+      var len = healthCenters.length;
+      var icon = L.icon ({iconUrl: 'C:/Users/Li/cpln692-week2/assignment/images/marker-icon.png',
```

Comment:
Absolute path URLs are best avoided because they make it difficult to use this code on anyone elses computer.

If you turn it relative, and do this:

`iconUrl: 'images/marker-icon.png'`

it works anywhere.